integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLivenessProbeCustomization.java

```diff
@@ -114,6 +126,11 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
 
     // create a basic model in image domain
     createAndVerifyMiiDomain(imageName);
+
+    // create temp file
+    String fileName = "tempFile";
```

Comment:
I'm not sure what the temp file is used for. Perhaps the name of the file can be changed to make its purpose more clear and/or comments should be updated.

Reply:
Updated the comments. This file is also used by other tests in the suite.